lib/tapjoy/ldap/cli/group/create.rb

```diff
@@ -0,0 +1,30 @@
+module Tapjoy
+  module LDAP
+    module CLI
+      module Group
+        # Create LDAP group
+        class Create
+          def create
+            # Check for errors
+            Trollop::die :type, "argument must be 'user' or 'service'" unless ['user', 'service'].include?opts[:type]
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put space between the method name and the first argument.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2634134584/issues/source?bid=3316970&fileBranchId=3318857#l9)